### PR TITLE
Release notes 1.12.2

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

File: CHANGELOG.md [hunk 1]
```diff
@@ -469,3 +470,19 @@ All notable changes to this project will be documented in this file. The format
...
+### Added
+ - AKO now claims support for Kubernetes 1.30, OCP 4.15
+
+### Changed
+ - AKO now creates single SSLKeyCertificate per tenant for default secret present in the cluster.
+
+### Fixed
+ - Fix: AKO Gateway does not create virtual service if Gateway have multiple listeners with same host name.
+ - Fix: AKO Gateway container crashes when it boots up in NPL mode.
+ - Fix: AKO does not honour readines probe for pods when AKO boots up in NPL mode.
```
Comment (DixitAakash) [inline CHANGELOG.md:485]: typo : readines should be readiness

**Review by DixitAakash — COMMENTED** [review]
Just a small comment. Other changes look good.

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Review by apalsule — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -469,3 +470,20 @@ All notable changes to this project will be documented in this file. The format
...
+## AKO-1.12.2
+
+### Added
+ - AKO now claims support for Kubernetes 1.30, OCP 4.15
+
+### Changed
+ - AKO now creates single SSLKeyCertificate per tenant for default secret present in the cluster.
+
+### Fixed
+ - Fix: AKO Gateway does not create virtual service if Gateway have multiple listeners with same host name.
```
Comment (abhishekbsingh) [inline CHANGELOG.md:483]: do you mean AKO Gateway container?

Comment (akshayhavile) [reply CHANGELOG.md:483]: Yes.  Is it creating any confusion?

**Review by DixitAakash — APPROVED** [review]

**Review by abhishekbsingh — APPROVED** [review]

**Comment by akshayhavile** [comment]
skip jira-id-check

**Pull request merged**